qiskit/quantum_info/synthesis/clifford_decompose.py

```diff
@@ -426,8 +433,10 @@ def _set_row_z_zero(clifford, circuit, qubit):
     and _set_row_x_zero has been called first
     """
 
-    x = clifford.stabilizer.X[qubit]
-    z = clifford.stabilizer.Z[qubit]
+    num_qubits = clifford.num_qubits
+    # get stabilizer
+    x = clifford.paulis.x[qubit + num_qubits]
+    z = clifford.paulis.z[qubit + num_qubits]
```

Comment:
maybe you can use here `stabilizer_tableau`  ?

Reply:
Sorry, I can't use it because `stabilizer_tableau` returns the copy but `paulis.z`returns the reference.

Reply:
OK, if there is a comment saying that these are the stabilizers then I think it's enough

Reply:
Sorry again. I noticed the `stabilizer_list()` method can be used here. The code is now more readable.